gramps/gui/views/treemodels/placemodel.py

```diff
@@ -139,7 +139,7 @@ def column_name(self, data):
     def search_name(self, data):
         """The search name includes all alt names to enable finding by alt name"""
         return ",".join(
-            [data["name"]["value"]] + [name["value"] for name in data["alt_name"]]
+            [data["name"]["value"]] + [name["value"] for name in data["alt_names"]]
```

Comment:
```suggestion
            [data.name.value] + [name["value"] for name in data.alt_names]
```
Do we want to update code use attributes when it is being edited anyway?

Reply:
All of the models seem to use dictionary access, but attributes would be nicer.  Is there any reason why we don't use them?

Reply:
Personally, I'd like to see code upgraded to use attributes when it is being edited.
You were too quick with this PR, creating it and merging before I could get my comment submitted!

Reply:
This PR was only a bug fix.  I made the smallest change possible.